### PR TITLE
Private and Experimental registry values

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -1991,21 +1991,21 @@ CachedInformationType Values" registry that was defined in [RFC7924].
...
+| 0xF0 - 0xFF | Reserved       | Private Use                              |
+|-------------+----------------+------------------------------------------|
 
 ### Expert Review guidelines
 
-The appointed Expert should ensure that the proposed hash algorithm is
-suitable for use as a cryptographic hash function, has a public specification
-and does not suffer from, or is known to be susceptible to, preimage attacks.
+The appointed Expert should ensure that the proposed hash algorithm has a public
+specification, neither suffers from nor is known to be susceptible to preimage
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Nit: One suggestion on the list was:
"Suitable for use as a cryptographic hash with no known preimage or collision attacks. These attacks can damage the integrity of the log."

I don't feel strongly about it.

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -1991,21 +1991,21 @@ CachedInformationType Values" registry that was defined in [RFC7924].
...
-| Index | Hash                |
-|-------+---------------------|
-| 0     | SHA-256 [RFC4634]   |
-| 255   | reserved            |
-|-------+---------------------|
-
-An expert review for adding values to this registry, to comply with the
-"Expert Review" policy described in [RFC5226].
+|-------------+----------------+------------------------------------------|
+| Value       | Hash Algorithm | Reference / [RFC5226] Assignment Policy  |
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Nit: Move the [RFC5226] reference from the column header and into the beginning of this section, saying something along the lines of:
"The Specification Required and Expert Review criteria mentioned in this section refer to the policies specified in [RFC5226]".

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -2024,26 +2024,33 @@ otherwise considered suitable for use as a cryptographic signature algorithm.
...
-| 7     | inclusion_proof_v2        |
-| 8     | x509_sct_with_proof_v2    |
-| 9     | precert_sct_with_proof_v2 |
-| 65535 | reserved                  |
-|-------+---------------------------|
+|-----------------+---------------------------+------------------------------------------|
+| Value           | Type and Version          | Reference / RFC5226 Assignment Policy    |
+|-----------------+---------------------------+------------------------------------------|
+| 0x0000          | Reserved                  | [RFC6962] (*)                            |
+| 0x0001          | x509_entry_v2             | RFCXXXX                                  |
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Replace RFCXXXX with "This Document" ? 
Not sure we'll get a chance to replace the placeholder with the actual number.

**Comment by eranmes** [comment]
Overall LGTM, I've made some suggestions on individual commits.

File: draft-ietf-trans-rfc6962-bis.md (hunk 2 above)
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/24122a90cfe8fc7e62c958c52adf76a35552bde8

File: draft-ietf-trans-rfc6962-bis.md (hunk 1 above)
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/3bc366c3f4d5dc327cc5e51e4474b3df249c4829

I also updated the equivalent text for signature algorithms in a similar fashion.

**Comment by robstradling** [comment]
@eranmes PTAL

File: draft-ietf-trans-rfc6962-bis.md (hunk 3 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: SGTM then.

**Pull request merged**